src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -597,6 +618,7 @@ protected override void OnNodePropertyChanged(object sender, PropertyChangedEven
             {
                 case "CachedValue":
                     Debug.WriteLine(string.Format("Requesting render packages for {0}", node.GUID));
```

Comment:
We would still need to remove geometry in `GenerateViewGeometryFromRenderPackagesAndRequestUpdate` in the call to `RemoveGeometryForUpdatedPackages` in cases where manipulator geometry needs updating for instance since the gizmo redraw is not always dependent on node update.